editor/css/editable-css.css

```diff
@@ -15,6 +15,10 @@
   }
 }
 
+.editor-wrapper.hidden {
+  display: none;
+}
```

Comment:
Is this really necessary, since `.hidden` is already defined in _common.css_:

https://github.com/mdn/bob/blob/eef4321c2793ef64be0b6a1f08340ca4a655094f/editor/css/editor-libs/common.css#L519-L521

Reply:
It's done because:
```
.hidden {
  display: none;
}
```
is always overridden by:
```
.editor-wrapper {
  display: flex;
  flex-direction: column;
}
```
Specificity is the same, but `.editor-wrapper` is in different file and has lower position.

I was thinking to:
- Move `.hidden` to editable-css.css, but then I would also need to copy it into editable-js-and-wat.css & tabbed-editor.css to maintain consistency.
- Make common.css be sent to the user after editable-css.css, but this could cause some unintended consequences.
- Use !important, @layer or a bit different selector, but current solution seems to me the cleanest.

Reply:
I do feel like adding `!important` to the `.hidden` rule is warranted in this case, because I cannot think of a reason why a "hidden" element should actually be shown.

Reply:
Alright, it's done :)